src/parser/utils.ts

```diff
@@ -27,3 +27,23 @@ export function isMap(d: unknown): d is Map<unknown, unknown> {
 export function isDate(d: unknown): d is Date {
   return Object.prototype.toString.call(d) === '[object Date]';
 }
+
+/** @internal */
+export type StylizeFunction = (x: string, style: string) => string;
+export type InspectParameterFn = (x: unknown, options: unknown) => string;
+// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+export const basicInspectParameterFn: InspectParameterFn = v => `${v}`;
+
+export function getStylizeFunction(options?: unknown): StylizeFunction {
```

Comment:
This is only used in regexp, right?  Maybe we could move this function out of the utils and into that file.

Reply:
Sounds good, I'll move it into the regexp function. My original reasoning was that we may want to stylize other types in special cases in the future, so it's best to have the stylize function available everywhere.

Reply:
Yeah, I’d agree with @aditi-khare-mongoDB and also slightly lean towards keeping it in the utils file, I know it’s not a great thing to have a “utils” file/directory/package/etc. where lots of super random stuff ends up, but the function is really not very regexp-specific either :slightly_smiling_face: